pkg/gatewayapi/supportedfeatures.go

```diff
@@ -0,0 +1,51 @@
+package gatewayapi
+
+import (
+	"errors"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"sigs.k8s.io/gateway-api/pkg/features"
+
+	"github.com/kong/gateway-operator/pkg/consts"
+)
+
+var (
+	traditionalCompatibleRouterSupportedFeatures = commonSupportedFeatures.Clone().Insert(
+	// add here the traditional compatible router specific features
+	)
+
+	expressionsRouterSupportedFeatures = commonSupportedFeatures.Clone().Insert(
+		// extended
+		features.SupportHTTPRouteMethodMatching,
+		features.SupportHTTPRouteQueryParamMatching,
+	)
+)
+
+var (
+	commonSupportedFeatures = sets.New(
+		// core features
+		features.SupportHTTPRoute,
+		features.SupportGateway,
+		features.SupportReferenceGrant,
+
+		// Gateway extended
+		features.SupportGatewayPort8080,
+
+		// HTTPRoute extended
+		features.SupportHTTPRouteResponseHeaderModification,
+		features.SupportHTTPRoutePathRewrite,
+		features.SupportHTTPRouteHostRewrite,
+	)
+)
```

Comment:
Can we reuse this in conformance tests in https://github.com/Kong/gateway-operator/blob/229786e24a261542b4e171886323d0cd97394f13/test/conformance/conformance_test.go#L39-L64 to prevent duplication?

Reply:
It's already part of this PR. Can't you see it?

Reply:
Missed it, sorry.